src/components/CropImageModal/CropImageModal.test.tsx

```diff
@@ -28,6 +24,26 @@ jest.doMock('react-avatar-editor', () => ({
     }),
 }))
 
+class FakeImg {
+    onload: () => void
```

Comment:
Shouldn't it be

```ts
onload: undefined | (() => void)
```

?

Reply:
it's just a mock class so I don't pay so much attention to typing here :)